### PR TITLE
Marcos.reginato/recycler view

### PR DESCRIPTION
**Pull request by regmarcos**
Hi guys! In this PR i made the Use Case for getting all the characters from the Marvel's API. As an example, i made a button that shows a Toast with the GSON received from the API.

This gif shows the Toast working.
pr1 mvvm

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -39,15 +40,20 @@ dependencies {
...
-    implementation "org.koin:koin-androidx-viewmodel:2.0.0-GA"
+    implementation appDependencies.coroutinesAndroid,
+            appDependencies.coroutinesCore,
+            appDependencies.viewModel,
+            appDependencies.ktx,
+            appDependencies.constraintLayout,
+            appDependencies.recyclerView,
+            appDependencies.glide,
+            appDependencies.cardView
+            //appDependencies.annotationProcessor
```
Comment (MuroSar) [inline app/build.gradle]: if this is not needed please delete it

File: app/src/main/res/layout/adapter_card_view_characters.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="200dp"
```
Comment (MuroSar) [inline app/src/main/res/layout/adapter_card_view_characters.xml]: no hardcode pls, use dimens

Comment (MuroSar) [inline app/src/main/res/layout/adapter_card_view_characters.xml]: no hardcode pls, use dimens

**Review by MuroSar — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (regmarcos) [reply app/build.gradle]: done

File: app/src/main/res/layout/adapter_card_view_characters.xml (hunk 2 above)
Comment (regmarcos) [reply app/src/main/res/layout/adapter_card_view_characters.xml]: done

Comment (regmarcos) [reply app/src/main/res/layout/adapter_card_view_characters.xml]: done

**Review by oadussang — APPROVED** [review]

**Review by MuroSar — APPROVED** [review]

**Pull request merged**